### ISSUE TITLE
Add "Comments" field

### ISSUE THREAD

**Issue by chowbok** [MediaBrowser/Emby#3476]
Can a "Comments" field be added to the metadata for information that doesn't fit in the existing fields?

**Comment by LukePulverenti** [comment]
Curious of the use case. What kind of things would you plan to comment on?

**Comment by chowbok** [comment]
I guess for me it would mainly be the provenance of the file; some movies have Blu-ray releases by different companies, I'd like to be able to note which one I'm using. I have movies where I merged the commentary track from a different edition; that'd be another thing I'd like to indicate and really can't with the existing metadata schema. Or if a movie cuts off or there's some other problem with the print. Also, there's currently no equivalent to IMDB's "movie connections" field; this could serve that function by listing sequels or remakes (even if they wouldn't be linked). Finally, this would be an acceptable workaround for Bug #3331 until that gets implemented.

These are just off the top of my head; I'm sure other people would find other uses for them.

**Comment by LukePulverenti** [comment]
The reason I'm asking is because if this is something that you could do with tags instead, then that be more powerful because you can filter based on tags to find other titles of the same criteria. And I would expect that tags will be incorporated into search in the future, as well as being able to see lists of tags.

**Comment by chowbok** [comment]
Right, I deliberately didn't list things that would be better served as tags. Something like "movies that take place in Chicago" would be a fine tag; "I copied the commentary from the Lion's Gate DVD and added it to the video from the Sony Blu-ray" probably wouldn't be.
